tests/manage/z_cluster/nodes/test_node_replacement_proactive.py

```diff
@@ -69,9 +69,12 @@ def check_node_replacement_verification_steps(
             f"of osd nodes. Wait for the new created worker node to appear in the osd nodes"
         )
         timeout = 1500
-        # In vSphere platform, we are creating new node with same name as deleted
+        # In vSphere UPI platform, we are creating new node with same name as deleted
```

Comment:
The PR title says IPI, but the comment here says UPI

Reply:
The comment belongs to https://github.com/red-hat-storage/ocs-ci/pull/7510, which was applicable only for Vmware UPI.